spec/views/posts/index_html_erb_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system, js: true do
```

Comment:
[OPTIONAL]
- If you want to check if all the tests are passing on your terminal, you can remove `...type: :system, js: true` and replace all that with `type: :feature`. 👍

Reply:
@Lucas-Masaba Thank you, I will update the code according to this.